services/web/client/source/class/osparc/component/workbench/EdgeUI.js

```diff
@@ -59,6 +64,28 @@ qx.Class.define("osparc.component.workbench.EdgeUI", {
   },
 
   members: {
+    __updateCurveColor: function() {
+      const modified = this.getEdge().getInputNode().getStatus()
+        .getModified();
+      let newColor = null;
+      if (modified === null) {
+        newColor = qx.theme.manager.Color.getInstance().resolve("workbench-edge-comp-active");
+      } else {
+        newColor = osparc.utils.StatusUI.getColor(modified ? "failed" : "ready");
```

Comment:
ooh... I see my request makes things a bit more complicated

Reply:
You are right. If there is no output at all, it can't be outdated or modified 👍 

What ```modified``` means is that the output that is being shown doesn't correspond to the current pipeline inputs/settings. So no output should be colored as gray.

Reply:
Done :)